domain/src/test/java/org/oppia/android/domain/exploration/ExplorationProgressControllerTest.kt

```diff
@@ -166,7 +189,13 @@ class ExplorationProgressControllerTest {
   @Test
   fun testPlayExploration_invalid_returnsSuccess() {
     val resultLiveData =
-      explorationDataController.startPlayingExploration("invalid_exp_id")
+      explorationDataController.startPlayingExploration(
+        profileId.internalId,
+        "invalid_topic_id",
```

Comment:
can we have some constants for these values
invalid_topic_id,invalid_story_id, invalid_exp_id

Reply:
done